ethpm/package.py

```diff
@@ -80,6 +74,37 @@ def get_contract_factory(self, name: ContractName, w3: Web3 = None) -> Contract:
         contract_factory = current_w3.eth.contract(**contract_kwargs)
         return contract_factory
 
+    def get_contract_instance(
+        self, name: ContractName, address: Address, w3: Web3 = None
+    ) -> Contract:
+        """
+        Return a Contract object representing the contract type at the provided address.
+        """
+        validate_contract_name(name)
+        current_w3 = self.get_w3_instance(w3)
+
+        try:
+            abi = self.package_data["contract_types"][name]["abi"]
+        except KeyError:
+            raise InsufficientAssetsError(
+                "Package does not have the ABI required to generate a contract instance "
+                "for contract: {0} at address: {1}.".format(name, address)
+            )
+        contract_instance = current_w3.eth.contract(address=address, abi=abi)
```

Comment:
It would be good to include the `binary` and `runtime_binary` in the constructor kwargs if they are available in the provided package assets.